es-ds-components/components/es-pagination.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+    <Paginator
```

Comment:
Nit: We don't yet have linting set up, but can we switch this to kebab-case like the rest of the components? i.e. `<paginator>`